test/appium/tests/base_test_case.py

```diff
@@ -67,7 +67,7 @@ def get_capabilities_sauce_lab():
     caps['appium:app'] = 'sauce-storage:' + test_suite_data.apk_name
     caps['appium:deviceName'] = 'Android GoogleAPI Emulator'
     caps['appium:deviceOrientation'] = 'portrait'
-    caps['appium:platformVersion'] = '10.0'
+    caps['appium:platformVersion'] = '12.0'
```

Comment:
I mean here. Let it be 10 for now

Reply:
sure, we did it just to test, I'll revert this commit

Reply:
reverted this change.